### PR TITLE
better client-events

### PR DESCRIPTION
**Pull request by barkanido**
* support a chain of completions in ClientEvents instead of only a single one.
* update Java client minor version

### CODE REVIEW

**Comment by haski** [comment]
look good.

File: src/aerospike_clj/client.clj [hunk 1]
```diff
@@ -74,7 +74,7 @@
                                   :el event-loops
                                   :dbns aero-ns
                                   :cluster-name cluster-name
-                                  :client-events (:client-events conf)}))))
+                                  :client-events (utils/vectorize (:client-events conf))}))))
```
Comment (yaronel) [inline src/aerospike_clj/client.clj:78]: The docstring needs to be updated to show it expects either a single `ClientEvents` object or a sequence of objects.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+#### VERSION 0.6.0
+* support ClientEvents vector to be a vector of copmletions onstead of a single one.
```
Comment (yaronel) [inline CHANGELOG.md]: support -> Support
copmletions -> completions
onstead -> instead

**Review by yaronel — COMMENTED** [review]
LGTM - A couple of small fixes you missed.

File: src/aerospike_clj/client.clj (hunk 1 above)
Comment (barkanido) [reply src/aerospike_clj/client.clj:78]: right.

File: CHANGELOG.md (hunk 2 above)
Comment (barkanido) [reply CHANGELOG.md]: my average typos per row.

**Review by yaronel — APPROVED** [review]

**Pull request merged**